# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../libarena/libarena-0.3.4-install.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Description: Running 'make install' fails with the following error message:

make: Circular /usr/lib/libarena.a <- /usr/lib/libarena.a dependency dropped.


--- libarena-0.3.4/src/Rules.mk.orig	2011-03-13 11:59:30.104072238 +0100
+++ libarena-0.3.4/src/Rules.mk	2011-03-13 12:01:41.692072238 +0100
@@ -93,14 +93,14 @@
 		$(LIBTOOL) --mode=install $(INSTALL_DATA) $(^:.a=.la) $(>:.a=.la) $(@:.a=.la); \
 	fi
 
-.INTERMEDIATE: -larena
+.INTERMEDIATE: larena
 
--larena: $(DESTDIR)$(libdir)/libarena.a $(DESTDIR)$(includedir)/arena/arena.h \
+larena:  $(DESTDIR)$(libdir)/libarena.a $(DESTDIR)$(includedir)/arena/arena.h \
          $(DESTDIR)$(includedir)/arena/pool.h $(DESTDIR)$(includedir)/arena/proto.h \
          $(DESTDIR)$(includedir)/arena/rbits.h $(DESTDIR)$(includedir)/arena/util.h \
          $(DESTDIR)$(includedir)/arena/align.h
 
-install: -larena
+install: larena
 
 
 #
